lib/ii.c

```diff
@@ -6550,6 +6591,8 @@ grn_ii_delete_one(grn_ctx *ctx, grn_ii *ii, grn_id tid, grn_ii_updspec *u, grn_h
   buffer_rec *br;
   buffer_term *bt;
   uint32_t pseg, size, *a;
+
+  GRN_SLOW_LOG_PUSH(ctx, GRN_LOG_DEBUG);
```

Comment:
This may be unbalanced because of `return ctx->rc`.

Reply:
Fixed. (but not elegant...)